### DISCUSSION TITLE
skywalking Linux Monitoring problem(version 9.0.0)

### DISCUSSION THREAD

**Discussion by kevinduan1984** [apache/skywalking#9665] [Q&A]
i have one problem about linux monitoring in Infrastructure.
i have followed the guide https://skywalking.apache.org/docs/main/v9.0.0/en/setup/backend/backend-vm-monitoring/ for config.
the skywalking version is 9.0.0, storage is elasticsearch 8.4.1. 
i have find  OpenTelemetry Collector already send the metrics to OAP server(the port is 11800), and in OAP server i can capture the packets that the OpenTelemetry Collector send. but now in elasticsearch there is nothing. 
the capture image, the oap server ip address is 192.168.0.240
![image](https://user-images.githubusercontent.com/114134877/191675684-597389b8-5571-4b1b-ad0c-8dc2262017e4.png)
the configuration about application.yml:
![image](https://user-images.githubusercontent.com/114134877/191676287-38960b04-ff0e-4c19-b09e-9d54f8e29f74.png)
the timezone in OAP server is +0800, but the meterics that send by opentelemetry collector is UTC, the image like this:
![image](https://user-images.githubusercontent.com/114134877/191677168-2b44aa82-7cca-41ab-947f-95aae32df2ed.png)
how can i solve this problem, thank you very much.

**Comment by wu-sheng** [comment]
Follow the showcase, I think once you have a k8s, you could try this out of box. https://skywalking.apache.org/docs/skywalking-showcase/next/readme/

> the timezone in OAP server is +0800, but the meterics that send by opentelemetry collector is UTC, the image like this:

UTC is just how logs print. If the time is right, OAP would covert this to server-side timezone. Also, TZ issue would only make metrics showing 8 hours before, you could check `last 1 week` metrics to see whether it is there. You mentioned `there is nothing in ElasticSearch`, if really so, there is nothing relative to timezone.

**Reply by kevinduan1984** [reply]
I think the show case can't solve my problem, can have some troubleshooting way to do? Thanks.

**Reply by wu-sheng** [reply]
> I think the show case can't solve my problem

How an env doesn't help you to cross-check? I can't follow your points.

I can't see what we can do. You even don't provide the error analysis. Whether OAP received the data? I can't guess that from no detail.

**Reply by kevinduan1984** [reply]
First picture in this discussion shows OAP get the packets.
i have checked skywalking logfiles and linux messages files there is no error in.
Can have other way to check? Thanks

**Reply by wu-sheng** [reply]
Have you even checked the data in ElasticSearch? The first graph is only showing traffic, what it posts is unknown. Don't treat it as same as you received expected data.

**Reply by kevinduan1984** [reply]
Yeah, no metrics data in ES, ES even can't creat meter_vm_** index.

**Reply by wu-sheng** [reply]
Then your OAP even doesn't boot successfully. That is what I said, don't treat your first screenshots. Read logs

**Reply by kevinduan1984** [reply]
Thanks, i will check logs again.

**Reply by kevinduan1984** [reply]
skywalking log file has one WARN log, the detail information as below showing:
2022-09-23 11:23:35,969 - org.apache.skywalking.oap.server.library.module.BootstrapFlow - 43 [main] INFO  [] - start the provider graphql in query module.
2022-09-23 11:23:36,476 - graphql.kickstart.tools.SchemaClassScanner - 355 [main] WARN  [] - Cannot find definition for field 'scope: Scope' on input type 'Entity' -> org.apache.skywalking.oap.server.core.query.input.Entity. Try adding it manually to the dictionary
the attachment is full logs after restart skywalking.
[skywalking startup logs.log](https://github.com/apache/skywalking/files/9642491/skywalking.startup.logs.log)
